config/characterise.py

```diff
@@ -1,19 +1,22 @@
-'''
-Override the default characterise config parameters by putting them in here.
-e.g.:
-config.doWrite = False
-'''
 # Cosmic ray removal
 # See: https://github.com/lsst/meas_algorithms/blob/master/src/CR.cc
 config.repair.doCosmicRay = True
 
-# We seem to be getting too many CR detections
-# For now we can increase the max number of allowable CRs
+# This controls the maximum number of allowable CRs in an image
 config.repair.cosmicray.nCrPixelMax = 1000000
 
-# Sometimes there are no initial source detections, causing errors
-# Lowering this number gives more sources for PSF fitting (default 10)
-config.detection.includeThresholdMultiplier = 5.0
+# Raising this contrast parameter reduces the number of CRs
+config.repair.cosmicray.cond3_fac = 10
 
-# Try and get PSF fitting to work
-config.psfIterations = 1 #More than this and it fails... not sure why.
+# Lowering this contrast parameter reduces the number of CRs
+config.repair.cosmicray.cond3_fac2 = 0
+
+
+# PSF settings
+# See: https://github.com/lsst/pipe_tasks/blob/master/python/lsst/pipe/tasks/measurePsf.py
+
+# Increasing this number gives a more accurate PSF estimate
+config.psfIterations = 3
```

Comment:
did you figure out what was causing it to fail if the number of iterations was more than 1?